donkeycar/parts/pose.py

```diff
@@ -0,0 +1,232 @@
+import logging
+import time
+from typing import Tuple
+
+from donkeycar.utils import is_number_type
+from donkeycar.parts.kinematics import NormalizeSteeringAngle, UnnormalizeSteeringAngle, TwoWheelSteeringThrottle
+from donkeycar.parts.kinematics import Unicycle, InverseUnicycle, UnicycleUnnormalizeAngularVelocity
+from donkeycar.parts.kinematics import Bicycle, InverseBicycle, BicycleUnnormalizeAngularVelocity
+from donkeycar.parts.kinematics import differential_steering
+from donkeycar.parts.tachometer import Tachometer, MockEncoder
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
```

Comment:
I think we should be setting the logging level only from `base.py` or `manage.py` so it cascades down and otherwise restrict hard-coding the level to a module's `main` routine.

Reply:
I agree. I removed the hard-coded log level.

Reply:
Seems to be some browser caching issue on my side, it's all good.